monkestation/code/modules/cassettes/cassette_approval.dm

```diff
@@ -78,17 +78,27 @@ GLOBAL_LIST_INIT(cassette_reviews, list())
 /datum/cassette_review/Destroy(force)
 	. = ..()
 	if(cassette_data)
-		QDEL_LIST(cassette_data)
+		cassette_data["side1"]["song_name"] = null
+		cassette_data["side1"]["song_url"] = null
+		cassette_data["side2"]["song_name"] = null
+		cassette_data["side2"]["song_url"] = null
+		cassette_data.Cut()
+		cassette_data = null
+	if(submitted_tape.loc == null)
+		QDEL_NULL(submitted_tape) // Remove any tapes in null_space. Denied or Pending condition.
+	submitted_ckey = null
```

Comment:
you don't have to set things like text or numbers to null btw, those don't harddel. only like references to datums/objects/mobs/clients/whatever